src/main/java/uk/gov/hmcts/reform/et/syaapi/exception/NotificationException.java

```diff
@@ -0,0 +1,14 @@
+package uk.gov.hmcts.reform.et.syaapi.exception;
+
+public class NotificationException extends RuntimeException {
+
+    private static final long serialVersionUID = -3042681110164047L;
```

Comment:
What is the reason for such a specific number for this? is this the minimum value for long? If so you can use long.min_value for easier to read code.

Reply:
Let me try updating as per your comment.